Phonebook.TwilightSaw/Controller/EmailController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using Microsoft.Extensions.Configuration;
+using MimeKit;
+using TwilightSaw.Phonebook.Model;
+using SmtpClient = MailKit.Net.Smtp.SmtpClient;
+
+namespace TwilightSaw.Phonebook.Controller
+{
+    public class EmailController()
+    {
+
+        public void SendEmail(Contact contact, string emailText, string head)
+        {
+            var configuration = new ConfigurationBuilder()
```

Comment:
🟠 **Dependency Injection**

💡 Can you not do this in `Program.cs` and inject it in the constructor?